build.sbt

```diff
@@ -164,7 +163,6 @@ lazy val semanticdbMetap = project.in(file("semanticdb/metap")).settings(
   sharedSettings,
   publishJVMSettings,
   fullCrossVersionSettings,
-  crossScalaVersions := LanguageVersions,
```

Comment:
so this uses all, as part of fullCrossVersionSettings?

Reply:
Yes, which makes sense since it compiles with full version suffix. And I think at some point I had issues with tests, which also use it and are cross compiled to all versions, but I had numerous issues at points, so not 100% sure if needed, but I do think the change is correct.